### ISSUE TITLE
defaulted value of INFLUX_MULT in keyword AQUANCON not being read by getSIdouble()

### ISSUE THREAD

**Issue by nairr** [OPM/opm-common#571]

**Comment by bska** [comment]
Immediate hunch: This is because the keyword [definition](https://github.com/OPM/opm-common/blob/f4a9298cf46f2b9cfff22d348857e38f09e777a1/src/opm/parser/eclipse/share/keywords/000_Eclipse100/A/AQUANCON#L12) uses `"default_value"` rather than `"default"`.

Does the following patch help?
---
```diff
diff --git a/src/opm/parser/eclipse/share/keywords/000_Eclipse100/A/AQUANCON b/src/opm/parser/eclipse/share/keywords/000_Eclipse100/A/AQUANCON
index d94c2afa..e0426a0b 100644
--- a/src/opm/parser/eclipse/share/keywords/000_Eclipse100/A/AQUANCON
+++ b/src/opm/parser/eclipse/share/keywords/000_Eclipse100/A/AQUANCON
@@ -9,7 +9,5 @@
  {"name" : "K2" , "value_type" : "INT"},
  {"name" : "FACE" , "value_type" : "STRING"},
  {"name" : "INFLUX_COEFF" , "value_type" : "DOUBLE" , "dimension" : "Length*Length"},
- {"name" : "INFLUX_MULT" , "value_type" : "DOUBLE" , "dimension" : "1" , "default_value" : 1.0},
- {"name" : "CONNECT_ADJOINING_ACTIVE_CELL" , "value_type" : "STRING" , "default_value" : "NO"}]}
- 
- 
+ {"name" : "INFLUX_MULT" , "value_type" : "DOUBLE" , "dimension" : "1" , "default" : 1.0},
+ {"name" : "CONNECT_ADJOINING_ACTIVE_CELL" , "value_type" : "STRING" , "default" : "NO"}]}
```

**Comment by nairr** [comment]
Yes, that was exactly the problem, works with the patch, will put in a PR for that. Thanks!

**Comment by joakim-hove** [comment]
It is a feature(?) that you can add what you want to the json without creating problems, but this it was unfortunate...

**Comment by bska** [comment]
I suppose this can be closed since #572 was merged into master?
